deepdream/README.md

```diff
@@ -0,0 +1,36 @@
+# Spell Workspace Example: deepDream
+
+This is an example of how to use Jupyter Notebook using Spell - adapted from https://github.com/google/deepdream
+
+For more information on deep dream, check out the blogpost https://ai.googleblog.com/2015/06/inceptionism-going-deeper-into-neural.html
+
+For more information regarding Spell, the Python API, runs, and workflows,
+check out the [Spell documentation](https://spell.run/docs).
+
+To run this code you first need to [sign up for a Spell account](https://web.spell.run/register)
+and install Spell: `pip install spell`.
+
+# Introduction
+Google's DeepDream is a computer vision program created by Alexander Mordvintsev which uses a CNN to find and enhance patterns in images.
+
+It uses the Caffe neural network framework trained on the ImageNet dataset to produce "dream visuals".
+
+We will use it today to attempt to turn a picture of dinosaurs into flowers (inspiration https://twitter.com/chrisrodley/status/875266719660482560)
+
+# Steps
+
+### 1. Create a Workspace
+Learn more at https://spell.run/docs/workspaces_overview
+
+Initialize from the git repo https://github.com/spell-examples/workspaces
```

Comment:
This looks like it might be a broken link. It probably needs to be `https://github.com/spellrun/spell-examples`. Maybe mention that the notebook location is in `deepdream/dream.ipynb` instead?